src/pandas_profiling/__init__.py

```diff
@@ -118,9 +182,9 @@ def get_rejected_variables(self) -> set:
             if message.message_type == MessageType.REJECTED
         }
 
-    def to_file(self, output_file: Path, silent: bool = True) -> None:
+    def to_file(self, output_file: Union[Path, str], silent: bool = True) -> None:
         """Write the report to a file.
-        
+
         By default a name is generated.
```

Comment:
The docstring is out-of-date. Please include .pp. You might also want to remove the out-of-date "By default a name is generated."